lib/components/base-components/NormalComponent.ts

```diff
@@ -189,7 +189,7 @@ export class NormalComponent<
     const { schematicSymbolName } = this.config
     if (!schematicSymbolName) return
     // TODO switch between horizontal and vertical based on schRotation
-    const symbol_name = `${this.config.schematicSymbolName}_horz`
+    const symbol_name = `${this.config.schematicSymbolName}`
 
     const symbol = (symbols as any)[symbol_name] as SchSymbol | undefined
```

Comment:
the errors in this file are basically just saying you need to go refactor every component that defined the symbol without orientation, which is fine.